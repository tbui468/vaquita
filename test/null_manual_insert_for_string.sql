create database sol;
open sol;

create table planets (name string, rings bool);
insert into planets (name, rings) values (null, false);
select * from planets;

close sol;
drop database sol;
exit;
