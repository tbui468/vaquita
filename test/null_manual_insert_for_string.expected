created database sol
opened database sol
created table planets
inserted 1 record(s) into planets
1, null, false
closed database sol
dropped database sol
